diff --git a/app/static/images/logos/11.jpg b/app/static/images/logos/11.jpg
old mode 100755
new mode 100644
index c4057ef..d01e74a
Binary files a/app/static/images/logos/11.jpg and b/app/static/images/logos/11.jpg differ
diff --git a/app/static/images/logos/17.jpg b/app/static/images/logos/17.jpg
old mode 100755
new mode 100644
index d64a125..49ef454
Binary files a/app/static/images/logos/17.jpg and b/app/static/images/logos/17.jpg differ
diff --git a/app/static/images/logos/24.jpg b/app/static/images/logos/24.jpg
old mode 100755
new mode 100644
index 96cc91b..174bd2e
Binary files a/app/static/images/logos/24.jpg and b/app/static/images/logos/24.jpg differ
diff --git a/app/templates/styles/footer.styl b/app/templates/styles/footer.styl
index 85de3cb..26d0776 100644
--- a/app/templates/styles/footer.styl
+++ b/app/templates/styles/footer.styl
@@ -1,16 +1,16 @@
-@import 'nib'
+// @import 'nib'
 
-$skew       = 25.62deg
-$height     = 150px
-$color-bg   = rgb(84,53,25)
-$color-line = rgb(255,153,51)
+// $skew       = 25.62deg
+// $height     = 150px
+// $color-bg   = rgb(84,53,25)
+// $color-line = rgb(255,153,51)
 
 
 
-&
-	height     : $height
-	background : $color-bg
-	position   : relative
+// &
+// 	height     : $height
+// 	background : $color-bg
+// 	position   : relative
 
 	
 	
diff --git a/app/templates/views/footer.jade b/app/templates/views/footer.jade
index 9dc9b5f..43c09fc 100644
--- a/app/templates/views/footer.jade
+++ b/app/templates/views/footer.jade
@@ -1,52 +1 @@
-#cover.row.collapse
-	#cover-rainbow
-		.color.color1
-		.color.color2
-		.color.color3
-		.color.color4
-		.color.color5
-		.color.color6
-.large-12.columns
-	label &nbsp;
-.large-2.columns.divcolor
-	img(src='/static/images/logo-white.png')
-.large-2.columns.divcolor1
-	.large-12.columns
-		.large-6.columns
-			a(href='https://twitter.com/remevet')
-				img.twitter(src='/static/images/logos/tw.png')
-
-			
-		.large-6.columns
-			a(href='https://www.facebook.com/remevet1')
-				img.facebook(src='/static/images/logos/fb.png')
-
-
-
-
-		
-
-.large-3.columns.divcolor2
-	.large-12.columns
-		.large-3.columns
-			img.size(src='/static/images/logos/mail.png')
-
-		.large-9.columns
-			h3 contacto@remevet.com
-	.large-12.columns.space
-		.large-3.columns
-			img.size(src='/static/images/logos/phone.png')
-		.large-9.columns.txtel
-			h3 (52)55 56892601
-			h3 (52)55 56899624
-	.large-12.columns
-
-		.large-3.columns
-			img.size(src='/static/images/logos/home.png')
-		.large-9.columns
-			h3 México D,F
-.large-5.columns.divcolor3
-	label &nbsp;
-
-.large-12.columns
-	label &nbsp;
+.large-12.small-12.medium-12.coluns
